src/core/dynamic_batch_scheduler.h

```diff
@@ -34,6 +34,7 @@
 #include <queue>
 #include <set>
 #include <thread>
+
```

Comment:
Remove extra lines between includes added here and elsewhere. Are you running the formatter? It should not be adding these lines.